### PR TITLE
Add/deploy script and codeowners

### PR DESCRIPTION
**Pull request by cmcarthur**

### CODE REVIEW

File: deploy.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+bundle install
+bundle exec middleman build
+aws s3 cp --recursive build/ s3://com.getdbt.hub/
```
Comment (chrispasakarnis) [inline deploy.sh:5]: Are files ever deleted/omitted via the build process? If so, I think just `cp`ing will leave the files that weren't rebuilt in the remote bucket

Comment (chrispasakarnis) [reply deploy.sh:5]: If so, `aws s3 sync --delete` might be a quick option, or always building into a new s3 path (prefix with some deploy identifier) and then switch the cloudfront distribution to read from the new prefix 

Neither need to hold us up today.

Comment (cmcarthur) [reply deploy.sh:5]: there's enough of a chance that the build fails and then sync deletes files that I don't want to use that. the semantics of this are such that files are rarely or never deleted. I'd rather clean that up down the road (and put a proper staging env in place to test)

Comment (cmcarthur) [reply deploy.sh:5]: I will add pipefail directives though so that this script exits early if the early build step fails

Comment (chrispasakarnis) [reply deploy.sh:5]: Roger

**Review by chrispasakarnis — APPROVED** [review]

**Pull request merged**